src/d3d11/d3d11_video.cpp

```diff
@@ -309,41 +309,16 @@ namespace dxvk {
           ID3D11Resource*         pResource,
     const D3D11_VIDEO_PROCESSOR_OUTPUT_VIEW_DESC& Desc)
   : D3D11DeviceChild<ID3D11VideoProcessorOutputView>(pDevice),
-    m_resource(pResource), m_desc(Desc), m_destructionNotifier(this) {
-    D3D11_COMMON_RESOURCE_DESC resourceDesc = { };
-    GetCommonResourceDesc(pResource, &resourceDesc);
-
-    DXGI_VK_FORMAT_INFO formatInfo = pDevice->LookupFormat(
-      resourceDesc.Format, DXGI_VK_FORMAT_MODE_COLOR);
-
-    DxvkImageViewKey viewInfo;
-    viewInfo.format = formatInfo.Format;
-    viewInfo.aspects = lookupFormatInfo(viewInfo.format)->aspectMask;
-    viewInfo.packedSwizzle = DxvkImageViewKey::packSwizzle(formatInfo.Swizzle);
-    viewInfo.usage = VK_IMAGE_USAGE_COLOR_ATTACHMENT_BIT;
-
-    switch (m_desc.ViewDimension) {
-      case D3D11_VPOV_DIMENSION_TEXTURE2D:
-        viewInfo.viewType   = VK_IMAGE_VIEW_TYPE_2D;
-        viewInfo.mipIndex   = m_desc.Texture2D.MipSlice;
-        viewInfo.mipCount   = 1;
-        viewInfo.layerIndex = 0;
-        viewInfo.layerCount = 1;
-        break;
-
-      case D3D11_VPOV_DIMENSION_TEXTURE2DARRAY:
-        viewInfo.viewType   = VK_IMAGE_VIEW_TYPE_2D_ARRAY;
-        viewInfo.mipIndex   = m_desc.Texture2DArray.MipSlice;
-        viewInfo.mipCount   = 1;
-        viewInfo.layerIndex = m_desc.Texture2DArray.FirstArraySlice;
-        viewInfo.layerCount = m_desc.Texture2DArray.ArraySize;
-        break;
-
-      case D3D11_VPOV_DIMENSION_UNKNOWN:
-        throw DxvkError("Invalid view dimension");
+    VideoProcessorView(pDevice, pResource, CreateViewInfo(Desc)),
+    m_desc(Desc), m_destructionNotifier(this) {
+    auto views = GetViews();
+    if (views[1] == nullptr) {
+      m_view = views[0];
+    } else {
+      DxvkImageViewKey viewInfo = m_viewInfo;
+      viewInfo.usage = VK_BUFFER_USAGE_TRANSFER_DST_BIT;
+      m_view = GetImage()->createView(viewInfo);
```

Comment:
As discussed earlier, this seems somewhat nonsensical since we can't actually do anything with a multi-planar "view"; for video views, `ClearView` needs to iterate over available aspects like everyone else.

If the game doesn't use `ClearView` then i'll happily take a `TODO` comment in that function as well for the time being as well since that requires reworking the entire `ClearView` implementation.